packages/web/src/components/collection/desktop/CollectionHeader.tsx

```diff
@@ -220,33 +277,77 @@ export const CollectionHeader = (props: CollectionHeaderProps) => {
               onPlay={onPlay}
               isPlayable={isPlayable}
               tracksLoading={tracksLoading}
+              isPremium={isPremium}
             />
           </ClientOnly>
-        </div>
+        </Flex>
         {onFilterChange ? (
-          <div className={styles.inputWrapper}>
-            <Input
+          <Flex
+            css={{
+              position: 'absolute',
+              top: spacing.l,
+              right: spacing.l,
+              width: '240px'
+            }}
+          >
+            <TextInput
+              label='Search in playlist...'
               placeholder={messages.filter}
-              prefix={<IconFilter color='subdued' />}
+              startIcon={IconSearch}
               onChange={handleFilterChange}
               value={filterText}
-              size='small'
-              variant='bordered'
+              size={TextInputSize.SMALL}
+              className={styles.searchInput}
             />
-          </div>
+          </Flex>
+        ) : null}
+      </Flex>
+
+      {/* Description section */}
+      <Flex
+        gap='xl'
+        direction='column'
+        p='xl'
+        backgroundColor='surface1'
+        borderTop='strong'
+        borderBottom='strong'
+      >
+        {isPremiumAlbumsEnabled && isStreamGated && streamConditions ? (
+          <GatedContentSection
+            isLoading={isLoading}
+            contentId={collectionId}
+            contentType={PurchaseableContentType.ALBUM}
+            streamConditions={streamConditions}
+            hasStreamAccess={hasStreamAccess}
+            isOwner={ownerId === currentUserId}
+            ownerId={ownerId}
+          />
         ) : null}
-      </div>
-      {isPremiumAlbumsEnabled && isStreamGated && streamConditions ? (
-        <GatedContentSection
-          isLoading={isLoading}
-          contentId={collectionId}
-          contentType={PurchaseableContentType.ALBUM}
-          streamConditions={streamConditions}
-          hasStreamAccess={hasStreamAccess}
-          isOwner={ownerId === currentUserId}
-          ownerId={ownerId}
-        />
-      ) : null}
+
+        <Flex className={cn(fadeIn)} gap='l' direction='column'>
+          {description ? (
+            <UserGeneratedText
+              size='xs'
+              className={cn(fadeIn)}
+              linkSource='collection page'
+              css={{ textAlign: 'left' }}
+            >
+              {description}
+            </UserGeneratedText>
+          ) : null}
+          <Flex>
```

Comment:
nit: should this be a `Box` or are we relying on the default flex props?

Reply:
Actually this didnt even need to be there at all 🙈